snakemake/Snakefile

```diff
@@ -121,13 +121,14 @@ rule guidescan_enumerate:
 
 
 rule add_ce:
```

Comment:
Please see this rule and some snakemake patterns used here - providing names to all inputs/outputs (so we don't have to do `input[1]` etc., which is unclear), and also specifying the input/output files directly as patters, instead of referring to other rules (which means the reader has to scroll to the relevant rule to find out what files are being talked about).

Perhaps we can use the same patterns for the rest of the rules, unless you disagree on something.